lib/ansible/modules/cloud/azure/azure_rm_functionapp_facts.py

```diff
@@ -122,6 +122,10 @@ def __init__(self):
             tags=dict(type='list'),
         )
 
+        required_if = [
```

Comment:
Are we able to push this through on a separate PR, I know it is a small change but we try to make each PR for a specific purpose.